### DISCUSSION TITLE
How to control keyboard dismissal?

### DISCUSSION THREAD

**Discussion by darren4flow** [kirillzyusko/react-native-keyboard-controller#360] [Q&A] [answered]
I'm building an app that renders a menu of buttons above the keyboard; however, the keyboard dismisses when I press on one of the menu buttons. This happens because the default behavior is to dismiss the keyboard if the user presses anywhere outside of a TextInput or keyboard, which means it doesn't even call the onPress function of the menu button.

I want to make it so that I can press on a button in the MenuBar without dismissing the keyboard. Can I do that with this package?

**Comment by kirillzyusko** [comment, marked as answer]
@darren4flow I think you can use https://reactnative.dev/docs/scrollview#keyboardshouldpersisttaps property to keep keyboard visible while tapping outside of the `TextInput`?

**Comment by darren4flow** [comment]
Thank you. This is exactly what I needed.
